docker/mongodb-kubernetes-tests/tests/authentication/helper_sharded_cluster_switch_project.py

```diff
@@ -0,0 +1,71 @@
+from kubetester import (
+    create_or_update_configmap,
+    read_configmap,
+)
+from kubetester.mongodb import MongoDB
+from kubetester.mongotester import ShardedClusterTester
+from kubetester.phase import Phase
+
+
+class ShardedClusterSwitchProjectHelper:
```

Comment:
How different is the implementation between ShardedClusterSwitchProjectHelper and ReplicaSetSwitchProjectHelper? Looks identical at first glance